.github/workflows/ci.yml

```diff
@@ -5,47 +5,42 @@ on:
     branches:
       - master
   pull_request:
-    branches:
-      - "*"
 
 jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - uses: dtolnay/rust-toolchain@nightly
+      - run: cargo test
 
   format:
     name: Format
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
Same as above.

Reply:
there are occasionally some edge cases that nightly rustfmt can resolve better than stable, though in general they're fairly consistent. The toolchain used for formatting doesnt impact the stability guarantees of the crate itself, so i guess it's a matter of choice. Do you prefer using stable or nightly compiler in your local dev environment?